lib/appRelationsDiscovery.js

```diff
@@ -0,0 +1,74 @@
+const {validate} = require('./utils');
+
+const supportedSyntax = ['default','mermaid'];
+
+const defaultOptions = {
+  syntax: 'default',
+};
+
+/**
+ * Validates and analyzes a list of asyncApiDocuments and get applications described by those files
+ * 
+ * @param {Array} asyncApiDocs An array of asyncApiDocuments
+ * @param {Object} syntax relation diagram format
+ * @returns {Promise<DiscoveredRelations>} Relations between documents in the requested syntax
```

Comment:
```suggestion
 * Validates and analyzes a list of AsyncAPI documents and get applications described by those files
 * 
 * @param {Array} asyncApiDocs An array of asyncApiDocuments
 * @param {Object} options for getting a relations
 * @returns {Promise<DiscoveredRelations>} Relations between documents
```

Better would be describe second argument as options, not as single syntax, and also better is write AsyncAPI as `AsyncAPI` not `asyncApi` etc :)